tools/src/code-review/index.ts

```diff
@@ -9,10 +9,25 @@ import reviewChangelogEntries from './reviewers/reviewChangelogEntries';
 import reviewForbiddenFiles from './reviewers/reviewForbiddenFiles';
 import { ReviewEvent, ReviewComment, ReviewInput, ReviewOutput, ReviewStatus } from './types';
 
+const DISABLE_CHANGELOG_MAGIC_COMMENT = '<!-- disable:changelog-checks -->';
+
 /**
  * An array with functions whose purpose is to check and review the diff.
  */
-const REVIEWERS = [checkMissingChangelogs, reviewChangelogEntries, reviewForbiddenFiles];
+const REVIEWERS = [
+  {
+    action: checkMissingChangelogs,
+    magicCommentToDisable: DISABLE_CHANGELOG_MAGIC_COMMENT,
+  },
+  {
+    action: reviewChangelogEntries,
+    magicCommentToDisable: DISABLE_CHANGELOG_MAGIC_COMMENT,
```

Comment:
Probably no need to disable this too, as it runs only when there are some changes in changelogs 😜